### PR TITLE
Update weebnation.py  find pls

### PR DESCRIPTION
**Pull request by KnuTNatioN**
If merged, please update the weebnation branch.

### CODE REVIEW

**Comment by y0sh1DE** [comment]
Gonna check this later

File: weebnation.py [hunk 1]
```diff
@@ -44,12 +46,17 @@ async def listAnimes(self, message):
...
+            msg = "Yo, damit kann ich nicht anfangen... viel zu wenig :rolling_eyes: "
         elif len(needle) >= 32:
-            await channel.send("😵: Das ist mir leider zu viel...")
-            return
+            msg = "Das ist mir zu viel zu suchen... :exploding_head: "
         else:
+            needle = "%" + needle + "%"
+            cmd = "SELECT aTitle, aLink FROM tblAnime WHERE aTags LIKE '%s' OR aTitle LIKE '%s'" % (needle, needle)
+            result = pymysql.executeSql(cmd)
+            print("found: ", len(result), "\n of max 3")
```
Comment (y0sh1DE) [inline weebnation.py:56]: (unneccessary) debug message

File: weebnation.py [hunk 2]
```diff
@@ -58,5 +65,10 @@ async def findAnime(self,message):
...
                 msg = "😁: Ich habe folgende Anime gefunden:\n"
                 for i in range(len(result)):
-                    msg = msg + ("%d. %s (%s) [%s]\n" % (i+1, result[i][0],result[i][1],result[i][2]))
-                await message.channel.send(msg)
\ No newline at end of file
+                    msg = msg + result[i][0] + " :link: " + result[i][1] + "\n"
+                    print(i)
+                    if i+2 > 3:
+                        break
+            else:
```
Comment (y0sh1DE) [inline weebnation.py:72]: else without if

**Review by y0sh1DE — CHANGES_REQUESTED** [review]
one error, one unneccessary line. apart from that lgtm

File: weebnation.py (hunk 1 above)
Comment (KnuTNatioN) [reply weebnation.py]: ok, killed the specific lines

File: weebnation.py (hunk 2 above)
Comment (KnuTNatioN) [reply weebnation.py:62]: what, in my push was no lonely else...
there is an
**if** ...stuff...
**elif** ...more stuff...
**else** ...end stuff...

**Pull request closed without merging**